src/python/doc/installation.rst

```diff
@@ -30,7 +30,8 @@ will be used by default, but you can force it by adding
 GUDHI Python module compilation
 ===============================
 
-To build the GUDHI Python module, run the following commands in a terminal:
+After making sure that the :ref:`Compilation dependencies` are properly installed,
+one can build the GUDHI Python module, by running the following commands in a terminal:
```

Comment:
(it isn't related to this PR, but while I am reading this text... Feel free to note that somewhere and ignore it for this PR)
Should we have a not explaining that if we build directly from a git checkout, it is `cd src/python`? Do we document user_version anywhere?

Reply:
It is true we do not talk about user and developer versions anywhere... Neither user_version

Reply:
I opened #310